joomla/joomla-cms
# Added tooltip of created date,created by and modified date fields at new/edit tag view

#### Summary of Changes

Check New/Edit tag view at administrator side.
In "Publishing" tag, there is no tool-tip for  Created Date, Created by and Modified date field.
After applying this PR, you can see tool-tip on for these fields.
#### Testing Instructions

Check New/Edit tag view at administrator side.
In "Publishing" tag, there is no tool-tip for  Created Date, Created by and Modified date field.
After applying this PR, you can see tool-tip on for these fields.
